APIServer/api_endpoints.py

```diff
@@ -224,12 +228,22 @@ class SourceCode(Resource):
     @api.response(HTTP_NOT_FOUND, 'Not Found')
     def get(self, model_id):
         """
-        Return the source code for a model. Not implemented yet.
+        Return the source code for a model.
         """
         model = get_model_by_id(model_id, indra_dir)
         if model is None:
             raise (wz.NotFound(f"Model {model_id} doesn't exist."))
-        return f"Feature to get source for {model_id} is coming soon!"
+
+        models = get_models(indra_dir)
+        file_name = None
+        for model in models:
+            if model.get('modelID') == model_id:
+                file_name = model.get('module') + '.py'
+
+        if file_name:
+            return requests.get(SOURCE_CODE_URL + file_name).json()
```

Comment:
The GitHub api response could fail.
Might be better to wrap this in try/except or check status code before returning